es-vue-base/src/lib-components/EsErrorPage.vue

```diff
@@ -0,0 +1,107 @@
+                href="mailto:hello@energysage.com">
+                <div class="circle mb-2">
+                    <div
+                        class="m-3 text-orange">
+                        <IconContactUs
+                            :width="iconFill"
+                            :height="iconFill" />
+                    </div>
+                </div>
+
+                <p :class="linkTextClass">Contact Us</p>
+            </a>
+            <a
+                :class="linkClass"
+                :style="linkStyle"
+                href="https://www.energysage.com/solar/">
+                <div class="circle mb-2">
+                    <div
+                        class="m-3 text-orange">
+                        <IconCommunitySolar
+                            :width="iconFill"
+                            :height="iconFill" />
+                    </div>
+                </div>
+                <p :class="linkTextClass">Learn About Solar</p>
+            </a>
+            <a
+                :class="linkClass"
+                :style="linkStyle"
+                href="https://news.energysage.com/">
+                <div class="circle mb-2">
+                    <div
+                        class="m-3 text-orange">
+                        <IconInfoBlogPost
+                            :width="iconFill"
+                            :height="iconFill" />
+                    </div>
+                </div>
+                <p :class="linkTextClass">Informative Blog Posts</p>
+
+            </a>
+            <a
+                :class="linkClass"
+                :style="linkStyle"
+                href="https://www.energysage.com/market-intro/">
+                <div class="circle mb-2">
+                    <div
+                        class="m-3 text-orange">
+                        <IconMarketplace
+                            :width="iconFill"
+                            :height="iconFill" />
+                    </div>
+                </div>
+                <p :class="linkTextClass">EnergySage Marketplace</p>
+            </a>
+        </div>
+    </div>
+</template>
+
+<script lang="js">
+import IconCommunitySolar from '../lib-icons/icon-community-solar.vue';
+import IconContactUs from '../lib-icons/icon-contact-us.vue';
+import IconMarketplace from '../lib-icons/icon-marketplace.vue';
+import IconInfoBlogPost from '../lib-icons/icon-info-blog-post.vue';
+
+export default {
+    name: 'EsErrorPage',
+    components: {
+        IconCommunitySolar,
+        IconContactUs,
+        IconMarketplace,
+        IconInfoBlogPost,
+    },
+    data() {
+        return {
+            linkClass: 'col-6 col-md-3 text-center',
+            linkTextClass: 'font-weight-bolder mb-4 mb-md-0',
+            linkStyle: 'display:flex; align-items: center; flex-direction:column',
+            iconFill: '100%',
+        };
+    },
+};
+</script>
+<style lang="scss" scoped>
+@import '~@energysage/es-bs-base/scss/includes';
+
+.circle {
```

Comment:
in the same vein of repeated styles, most of these can be bootstrap utility classes (i.e. `align-items-center`; `bg-gray-100`; `border-circle`).  Leaning towards putting those as classes and leaving css as minimal as possible